www/js/control/SettingRow.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { angularize } from "../angular-react-helper";
+import { Text } from 'react-native-paper';
+import { useTranslation } from "react-i18next";
+import { string, func, object} from "prop-types";
+
+/*
+biggest struggle here is how to pass i18n keys as parameters (or strings in general)
+*/
+
+const SettingRow = ({textKey, iconName}) => {
+    const { t } = useTranslation(); //this accesses the translations
+    console.log("row created!!!!!", "translate hard-code" + t('general-settings.are-you-sure') + "translate via param" + t(textKey)+ " and the icon (plain string) " + iconName);
```

Comment:
@JGreenlee this log statement highlights the issue I've been having (no rush, I'm about to be 98% offline till Tuesday anyways)